applications/FluidDynamicsApplication/python_scripts/biphasic_levelset_solver.py

```diff
@@ -2,15 +2,15 @@
 # -*- coding: utf-8 -*-
 # importing the Kratos Library
 from KratosMultiphysics import *
-# from KratosMultiphysics.ConvectionDiffusionApplication import *
 from KratosMultiphysics.ThermoMechanicalApplication import *
 from KratosMultiphysics.FluidDynamicsApplication import *
 from KratosMultiphysics.MeshingApplication import *
 from KratosMultiphysics.IncompressibleFluidApplication import EstimateDt3D
 
-import pureconvection_solver
-import thermal_solver
-import monolithic_solver_eulerian
+import KratosMultiphysics.ConvectionDiffusionApplication
+from KratosMultiphysics.ConvectionDiffusionApplication import pureconvection_solver
+from KratosMultiphysics.ConvectionDiffusionApplication import thermal_solver
+from KratosMultiphysics.ConvectionDiffusionApplication import monolithic_solver_eulerian
```

Comment:
All this imports belong to the convection diffusion, but was not imported so I re-activated it.